OpenSim/Simulation/Wrap/WrapObject.cpp

```diff
@@ -64,7 +64,8 @@ void WrapObject::constructProperties()
     constructProperty_quadrant("Unassigned");
     Appearance defaultAppearance;
     defaultAppearance.set_color(SimTK::Cyan);
-    defaultAppearance.set_representation(VisualRepresentation::DrawWireframe);
+    defaultAppearance.set_opacity(0.5);
+    defaultAppearance.set_representation(VisualRepresentation::DrawSurface);
```

Comment:
This is default appearance, users who cared and changed that will have their preserved AFAIK.  When updating the code to remove hardcoded settings, that broke the test cases. Based on the code, the API visualizer used to not set representation (default Surface) and a hard-coded opacity 0.5 while GUI used hard-coded (wire-frame) and no-opacity. Something has to give but I'm ok with changing to Wireframe and Opacity 0.5 and updating test cases accordingly. Please let me know your preference.